src/utils/flagHelper.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+import * as options from './options';
+
+const BOOLEAN_FLAGS = {
```

Comment:
I thought we discussed this before, that we couldnt have a hard coded list of boolean flags, because it's context dependent?